src/components/Article.js

```diff
@@ -43,22 +48,32 @@ export const Article = ({ lang, title }) => {
     window.location.hash = `/quickfacts/${lang}/${title}`
   }
 
-  if (!article) {
-    return 'Loading...'
-  }
-
   const hasImage = !!article.imageUrl
 
+  const pagerEvent = {
+    nextPage: () => {
+      const sectionLength = article.sections.length
+      const nextSection = currentSection + 1
+
+      if (nextSection < sectionLength) setCurrentSection(nextSection)
+      else setCurrentSection(0)
+    },
+    prevPage: () => {
+      const prevSection = currentSection - 1
+      if (prevSection >= 0) setCurrentSection(prevSection)
+    }
+  }
+
   return (
     <Fragment>
-      <Pager>
+      <Pager event={pagerEvent}>
         <div class='page article'>
           { hasImage && <div class='lead-image' style={{ backgroundImage: `url(${article.imageUrl})` }} /> }
```

Comment:
The lead image should change to be the first image found in each section.

Reply:
done in https://github.com/wikimedia/wikipedia-kaios/pull/15/commits/c15095c1867629dc484e528377e0187ef2c37a14